runbooks/enable-permissionless/bundle.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+OPTIMISM_PORTAL_PROXY=0x5d66c1782664115999c47c9fa5cd031f495d3e4f
+GUARDIAN=0x09f7150D8c019BeF34450d6920f6B3608ceFdAf2
+CHAINID=57073
+#DGM=0xc6901F65369FC59fC1B4D6D6bE7A2318Ff38dB5B
```

Comment:
This is a runbook (so it sounds like this should generalize) but we have hardcoded data in here for one chain, and the runbook README has no instructions to change these values. My suggestion would be to delete these four declarations, and in the bundle readme, including instructions for setting those env vars and having the bundle read them. It would also be ideal if we can simply only include chain ID, and fetch the portal and guardian addreses from the superchain registry from that.

We may want to split this up into two PRs (this runbook, which we can iterate on more, and the ink task) to avoid the ink task being blocked by the runbook comments

Reply:
Below in this file we also have hardcoded `data` and `signatures` which seems wrong. This script feels like it's not fully templatized?e

Reply:
@Inphi gave me this script and you're right its not fully templatized so I'm going to remove this